packages/components/src/systems/PharmacySearch/index.tsx

```diff
@@ -33,13 +33,28 @@ export default function PharmacySearch(props: PharmacyProps) {
         });
   });
 
+  const handleResetAddress = () => {
+    if (props?.setPharmacy) {
+      props.setPharmacy(undefined);
+    }
+    actions.clearPharmacies();
+  };
+
   createEffect(() => {
     if (selected()?.id) {
       setQuery('');
       props?.setPharmacy?.(selected());
     }
   });
 
+  const formattedAddress = (pharmacy: {
```

Comment:
Hard to tell, is this just refactoring? Seems unrelated but guessing its no real chnage

Reply:
Actually I reworked the `ComboBox` for `DoseCalculator` which affected this component.